File: Maximum Number of Coins You Can Get.cpp
class Solution {
public:
    int maxCoins(vector<int>& piles) {
        int n = piles.size();
        int sum = 0;
        sort(piles.begin(),piles.end(),greater());
        for(int j= 0 ,i = 1 ; j < n/3 ; i+=2,j++)
            sum = sum + piles[i];
        return sum;
    }
};

File: MaximumNumberOfCoinsYouCanGet.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        piles.sort(reverse = True)
        maxCoins = 0

        for index in range(1,len(piles) - len(piles)//3,2):
            maxCoins += piles[index]

        return maxCoins
